infrastructure/modules/ui/variables.tf

```diff
@@ -59,3 +59,9 @@ variable "geo_restriction_locations" {
   type        = list(string)
   default     = ["GB"]
 }
+
+variable "sql_injection_protection" {
+  description = "Wheather to add SQL injection protection to WAF."
```

Comment:
Spelling issue + would you be okay adding a bit of text to flag that this disables some rules in the WAF that add protection to the overall application. Ok with this is in principle, just want users to be aware of the risks.